src/python/TaskWorker/Actions/DBSDataDiscovery.py

```diff
@@ -434,22 +434,11 @@ def executeInternal(self, *args, **kwargs):
             secondaryBlocksWithLocation = secondaryLocationsMap.copy().keys()
 
         # filter out TAPE locations
-        # temporary hack until we have a new config. parameter: allow user to accpet a partial dataset
-        # by inserting '0' as (first element of) runRange
-        usePartialDataset = False
-        runRange = kwargs['task']['tm_split_args']['runs']
```

Comment:
Do not delete `runRange`. It is needed for [line 480](https://github.com/dmwm/CRABServer/blob/5bd657afce5387c95bfe830285fbda01cfc88e48/src/python/TaskWorker/Actions/DBSDataDiscovery.py#L480).